modules/knative-serving-advanced-config.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * /serverless/installing_serverless/serverless-install-config-options.adoc
+
+[id="knative-serving-advanced-config_{context}"]
+= Knative Serving supported installation configuration options
+
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+image::serving-form-view.png[Create Knative Serving form in web console Administrator Perspective]
+
+== Controller Custom Certs
```

Comment:
Can this be "Controller custom certificates" or does it need to be capitalized and "certs"?

Reply:
Also suggest adding a unique ID, e.g. `[id="knative-serving-controller-custom-certs_{context}"]`.

Reply:
I'd keep it like this for now just because that's how it's displayed on the console.
Let me look into getting this updated in the UI.